### PR TITLE
DSD-1792: Changes (required) text capitalization

### PR DESCRIPTION
**Pull request by oliviawongnyc**
Fixes JIRA ticket [DSD-1792](https://newyorkpubliclibrary.atlassian.net/browse/DSD-1792)

## This PR does the following:

- Lowercases the 'r' of '(required)' in labels and placeholders where necessary
- Updates documentation

## How has this been tested?

- Locally, in Storybook

## Accessibility concerns or updates

- None

### Accessibility Checklist

- [ ] Checked Storybook's "Accessibility" tab for color contrast and other issues.
- [ ] The feature works with keyboard inputs including tabbing back and forward and pressing space, enter, arrow, and esc keys.
- [ ] For hidden text or when `aria-live` is used, a screenreader was used to verify the text is read.
- [ ] For features that involve UI updates and focusing on DOM `ref`s, focus management was reviewed.
- [ ] The feature works when the page is zoomed in to 200% and 400%.

## Open Questions

-

### Checklist:

- [ ] I have updated the Storybook documentation and changelog accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

[DSD-1792]: https://newyorkpubliclibrary.atlassian.net/browse/DSD-1792?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 ### Updates
 
 - Updates `Tooltip` component with `placement` and `offset` props for custom positioning.
+- Lowercases the 'r' of '(required)' in labels and placeholders where necessary
```
Comment (EdwinGuzman) [inline CHANGELOG.md]: Can you add to the effect of "in order to align with sentence case guidelines".

**Review by EdwinGuzman — COMMENTED** [review]
Looks good just one minor detail

File: CHANGELOG.md (hunk 1 above)
Comment (oliviawongnyc) [reply CHANGELOG.md]: Done!

**Review by bigfishdesign13 — APPROVED** [review]
Looks great!  I did notice that the font weight for the `required` text does not match what we have in the VDL, but I am going to create a new ticket for that update rather than trying to tack it on here.

**Review by EdwinGuzman — APPROVED** [review]

**Comment by oliviawongnyc** [comment]
> Looks great! I did notice that the font weight for the `required` text does not match what we have in the VDL, but I am going to create a new ticket for that update rather than trying to tack it on here.

Sounds good @bigfishdesign13. Thank you!

**Pull request merged**